README.md

```diff
@@ -246,3 +246,12 @@ If you want to initialize the DB with production data, run the following command
 ```bash
 bin/console app:load-organizations
 ```
+
+If you want to load huge amount of data, for example to test the performances, run the following command in the fpm container:
```

Comment:
we should avoid running commands in the fpm container, as there is a dedicated one :

If you need to run a command with ENV vars :
```bash
bin/tools sh -c "APP_ENV=test bin/console"
```